package/firmware/ipq-wifi/Makefile

```diff
@@ -218,6 +219,7 @@ $(eval $(call generate-ipq-wifi-package,spectrum_sax1v1k,Spectrum SAX1V1K))
 $(eval $(call generate-ipq-wifi-package,tplink_eap610-outdoor,TPLink EAP610-Outdoor))
 $(eval $(call generate-ipq-wifi-package,tplink_eap620hd-v1,TP-Link EAP620 HD v1))
 $(eval $(call generate-ipq-wifi-package,tplink_eap623od-hd-v1,TP-Link EAP623-Outdoor HD v1))
+$(eval $(call generate-ipq-wifi-package,tplink_eap625-outdoor,TP-Link EAP625-Outdoor))
```

Comment:
Same here. `v1` can be optional. but better still have `HD`.

I still suggest adding v1 here. So we won't get into trouble when they release v2.